pubspec.yaml

```diff
@@ -1,10 +1,10 @@
 name: http_io
-version: 0.10.0-dev
+version: 0.10.0
 author: "Dart Team <misc@dartlang.org>"
 homepage: https://github.com/dart-lang/http_io
 description: HTTP Client and Server APIs.
 environment:
-  sdk: ">=1.24.0 <2.0.0"
+  sdk: ">=2.0.0-dev.17.0 <2.0.0"
```

Comment:
Oh, sorry, you did it here. Thanks!

Reply:
But we should probably also inc the package version and republish, I guess. Is that right, @kevmoo ?